op/projected_gradient.go

```diff
@@ -44,7 +44,6 @@ func (pg *ProjectedGradient) Minimize(loss Function, stop StopCriteria, vec Para
 
 	for k := 0; !stop.Done(stt, ovalgrad.value, ovalgrad.gradient); k += 1 {
 
-		pg.projector.ClipGradient(stt, ovalgrad.gradient)
```

Comment:
`stt` is always in the bounded region, so the gradient is never clipped. So in this commit we remove this line.